packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -168,7 +167,6 @@ require.config({
 define("$bootstrapModule", ["$entrypoint", "dart_sdk"], function(app, dart_sdk) {
   dart_sdk.dart.setStartAsyncSynchronously(true);
   dart_sdk._debugger.registerDevtoolsFormatter();
-  dart_sdk.dart.nonNullAsserts($nullAssertions);
```

Comment:
@nshahan is it ok that I deleted this line? Or should I leave this line and hard-code the value (since I deleted the CLI arg in this PR)?

Reply:
Yes, looks like good cleanup. This is totally unneeded as long as you are prevented from passing 
`--no-sound-null-safety` when compiling. It will be defaulted to `false` in the compiled SDK until it is fully cleaned up.

Reply:
Thanks!